tools_for_todoist/services/calendar_to_todoist.py

```diff
@@ -118,8 +118,16 @@ def _process_updated_event(self, old_calendar_event, calendar_event):
 
         if todoist_id is None:
             return None
-
         todoist_item = self.todoist.get_item_by_id(todoist_id)
+        if (
+            old_calendar_event is not None
+            and old_calendar_event.is_declined_by_me()
```

Comment:
for a recurring event, what is the meaning of the `attendees` list and declined, does it mean you have declined the entire sequence?